composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/AddSpotView.kt

```diff
@@ -135,122 +119,109 @@ fun AddSpotView(onBackClick: () -> Unit, onSpotAdded: () -> Unit, addSpotViewMod
-                        unfocusedLabelColor = Color.Gray,
-                        textColor = Color.DarkGray
-                    ),
+            AddSpotSearchBox(addSpotViewModel)
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            Column {
+                Box(
                     modifier = Modifier
-                        .fillMaxWidth()
                         .height(55.dp)
-                        .shadow(8.dp, RoundedCornerShape(12.dp))
+                        .shadow(12.dp, RoundedCornerShape(12.dp))
                         .background(color = Color.White, shape = RoundedCornerShape(12.dp))
                         .clip(RoundedCornerShape(12.dp))
-                )
-
-                LazyColumn(
-                    modifier = Modifier.fillMaxWidth()
-                        .shadow(8.dp, RoundedCornerShape(12.dp))
-                        .background(Color.White)
-                        .clip(RoundedCornerShape(12.dp))
                 ) {
-                    items(suggestions) { suggestion ->
-                        Text(
-                            text = suggestion.name,
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .padding(8.dp)
-                                .clickable {
-                                    scope.launch {
-                                        suggestionGenerator.fetch(suggestion)?.let { feature ->
-                                            addSpotViewModel.updateCurrentLocation(
-                                                Coordinate(
-                                                    feature.geometry.coordinates[0],
-                                                    feature.geometry.coordinates[1]
-                                                )
-                                            )
-                                        }
-                                    }
-                                    focusManager.clearFocus()
-                                    addSpotViewModel.updateQuery(suggestion.name)
-                                    addSpotViewModel.updateSuggestions(emptyList())
-                                    searchedSuggestion.value = suggestion
-                                },
-                            color = Color.DarkGray,
+                    Row(
+                        modifier = Modifier.padding(start = 16.dp, end = 8.dp),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Column {
+                            Text("Building Type", fontSize = 12.sp, color = Color.Gray)
+                            Text(selectedBuildingType)
+                        }
+
+                        Spacer(modifier = Modifier.weight(1f))
+
+                        IconToggleButton(
+                            checked = buildingTypeExpanded,
```

Comment:
Is there not a builtin dropdown menu?

Reply:
I did try to use that but it wasn't simple to format the list to resemble our other dropdown menus. I could try to mess around with it but might not be worth it for now.

Reply:
In that case, maybe making the whole field clickable would make it better.

Reply:
I updated it to be a clickable field now